src/components/widget/WidgetWindowFullScreen.tsx

```diff
@@ -0,0 +1,23 @@
+import Chat from './Chat';
+import { ChatAiWidgetProps } from './ChatAiWidget';
+import ProviderContainer from './ProviderContainer';
+import { elementIds } from '../../const';
+
+/**
+ * NOTE: External purpose only.
+ * Do not use this component directly. Use Chat instead for internal use.
+ */
+function WidgetWindowFullScreen(props: ChatAiWidgetProps) {
+  return (
+    <ProviderContainer {...props}>
+      <div
+        id={elementIds.widgetWindow}
+        style={{ position: 'fixed', left: 0, top: 0, width: '100%', height: '100%', transform: 'scale(1)' }}
```

Comment:
`scale(1)` is a trick to ensure that a modal with `position: fixed` renders correctly within the widget window.